src/commands/lambda/__tests__/flare.test.ts

```diff
@@ -66,6 +70,7 @@ const MOCK_LOG_GROUP = 'mockLogGroup'
 const MOCK_OUTPUT_EVENT: OutputLogEvent[] = [{timestamp: 123, message: 'Log 1'}]
 const MOCK_LOGS = new Map().set('log1', MOCK_OUTPUT_EVENT)
 const cloudWatchLogsClientMock = mockClient(CloudWatchLogsClient)
+const lambdaClientMock: any = mockClient(LambdaClient)
```

Comment:
Why are we typing as `any` here, but not in the CWL?

Reply:
If I remove the typing as `any`, then `await getTags(lambdaClientMock, MOCK_REGION, MOCK_ARN)` has a type error:
```
TS2345: Argument of type 'AwsStub<ServiceInputTypes, ServiceOutputTypes>' is not assignable to parameter of type 'LambdaClient'.
```

Typing it as `LambdaClient` instead of `any` gives the error:
```
TS2739: Type 'AwsStub ' is missing the following properties from type 'LambdaClient': config, destroy, middlewareStack
```

Reply:
I see, sounds about right, normally we pass it as `lambdaClientMock as any`, could we do that on the function in which you are passing it? Just to keep the typing consistent in cases where the `any` typing is not needed?

Reply:
Yes I'll pass it into functions as `lambdaClientMock as any` instead!

Reply:
I don't see the change 😯

Reply:
Oops, it's changed now